spotify/views.py

```diff
@@ -1,5 +1,6 @@
 from django.shortcuts import render, redirect, HttpResponse
 from django.http import JsonResponse
+from django.core.exceptions import PermissionDenied
```

Comment:
allows us to return permission denied when appropriate instead of generic 500 internal server error